app/static/css/style.css

```diff
@@ -115,35 +153,119 @@ footer {
 }
 
 .thin-footer {
-  font-family:'Nunito-Sans', sans-serif;
+  font-family:'Inter', sans-serif;
   font-weight: 100;
   letter-spacing: 0.5px;
   color: var(--grey-mid);
 }
 
+.expand {
+  transition: all ease 0.5s;
+}
+
+.expand:hover {
+  font-size: 101%;
```

Comment:
Does this 1% matter?

Reply:
yes, it's the small things that matter in awesome frontend!! ;)

Reply:
I shall assume that's because this gets rounded up to the next font size on your display. Please specify that instead of a percentage.

Reply:
What's the issue with using a percentage? It always gets rounded up. You want me to hard code the font size..? I can't really do font-size += 1 in css... the percentage is a good solution in my opinion.

Reply:
let me know, else I assume this is resolved..

Reply:
Okay, you're right in it increasing on (probably) all browsers because of more accurate font rendering. I would prefer an exact number, but this is okay.

---

I have one more issue which I'll leave here since it's related to animation of these links. Our links do so many weird things. The standard footer links turn a different colour, but don't animate. The "Kurius" link changes colour and has a bar that slides in from the left. The "HackItForward" link (why it's a link doesn't actually make sense to me either, it's a link from the site to itself, and it being a home page link doesn't make sense because there's one in the header and another in the footer already) expands in size unnaturally (it first increases by 1 pixel horizontally, then vertically, so it looks like there's a stutter) and changes colour?

Is there a reason for not sticking to something a little more consistent? I can understand important links having an animation, and the rest just changing colour, but 3 different kinds in only the footer is too much.

Reply:
you're right, I will remove the HackItForward as a link at the footer, as well as the expand effect. I also think the footer links shouldn't change colors on hover apart from the Kurius words, to emphasize that it is an external link.

I have a problem though, I can't seem to be able to override the behavior of not changing colors of links on hover. 
```html
<a href="/terms" class="grey">
<li class="list-item">Terms of Service</li>
</a>
```
Even if I hard code the color styling on hover, it doesn't seem to do anything. It only works if I don't wrap the words inside the <li> tag, like below, but the footer will look different now. 

```html
<a href="/terms" class="grey">
Terms of Service
</a>
```

been trying to figure this out for 30 minutes

Reply:
I think the colour change should be kept. It's the standard way (alongside the cursor) that indicates a link to the user. However, the reason what you're doing isn't working is because you need to override the `:hover` selector.

Reply:
This is the styling for hover.
```css
.grey {
  color: var(--grey-mid)
}

.grey:hover {
  color: var(--grey-mid);
}
```

Reply:
Not sure then, perhaps Cirrus is loaded after, this CSS isn't loaded for some reason, or the specificity calculation returns the wrong value.

Reply:
ok color changes are kept, i removed the hover grow effect